packages/yswaps/scripts/configs/fantom.ts

```diff
@@ -16,6 +17,7 @@ const fantomConfig: StrategyConfiguration<'FANTOM'> = {
           },
         ],
         solver: 'Dexes',
+        dustThreshold: utils.parseEther('250'),
```

Comment:
I think it might be wise to put `dustThreshold` inside enabledTrades ? Since we might have multiple `tokenIns` threhsolds might be different p/token in ? Does it make sense ?

Reply:
yeah i thought the same exact thing but didnt want to change it right now since looked overkill for now. Will take a look and see how much refactor requires.